client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -373,6 +373,18 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         SELENIUM_FIREFOX_PROFILE("firefoxProfile", "", false),
 
         /**
+         * Adds additional chromedriver chromeOptions args to be used with local/remote runs using the chrome
+         * browser. <br>
+         * The string value here is specified as a comma separated list of arguments. Arguments with
+         * associated value should be separated by a '=' sign.
+         *
+         * For e.g.
+         *
+         * "start-maximized, user-data-dir=/some/path"
+         */
+        SELENIUM_CHROME_OPTIONS("chromeOptions", "", false),
```

Comment:
Also anything comma separated will automatically be turned into a `List` by commons-config and calling `getConfigProperty` will only return the first value.